ios/StripeTerminalReactNative.swift

```diff
@@ -989,6 +989,38 @@ class StripeTerminalReactNative: RCTEventEmitter, DiscoveryDelegate, BluetoothRe
         self.collectRefundPaymentMethodCancelable = nil
     }
 
+    @objc(collectData:resolver:rejecter:)
+    func collectData(params: NSDictionary, resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) {
+        let collectDataTypeParam = params["collectDataType"] as? String ?? ""
+        let enableCustomerCancellation = params["enableCustomerCancellation"] as? Bool ?? false
+
+        let collectDataType = Mappers.mapToCollectDataType(collectDataTypeParam)
+        guard let collectDataType else {
+            resolve(Errors.createError(code: CommonErrorType.InvalidRequiredParameter, message: "You must provide correct collectDataType parameter."))
```

Comment:
would recommend keeping this the same as android
```suggestion
            resolve(Errors.createError(code: CommonErrorType.InvalidRequiredParameter, message: "You must provide a collectDataType"))
```